Buildings/Controls/OBC/CDL/Continuous/LimPID.mo

```diff
@@ -53,8 +51,6 @@ block LimPID
   parameter Real y_start=0 "Initial value of output"
     annotation(Dialog(enable=initType == CDL.Types.Init.InitialOutput, group=
           "Initialization"));
-  parameter Boolean strict=true "= true, if strict limits with noEvent(..)"
-    annotation (Evaluate=true, choices(checkBox=true), Dialog(tab="Advanced"));
   parameter Boolean reverseAction = false
     "Set to true for throttling the water flow rate through a cooling coil controller";
   parameter Buildings.Controls.OBC.CDL.Types.Reset reset = Buildings.Controls.OBC.CDL.Types.Reset.Disabled
```

Comment:
should the `Evaluate=true` be used here?

Reply:
Here `Evaluate=true` made sense as the user is not likely to want to change this between simulations, and it could affect the performance of the solver.